src/Module/EddBkRestApiModule.php

```diff
@@ -0,0 +1,528 @@
+                    return new UpdateBookingHandler($c->get('eddbk_bookings_controller'));
+                },
+
+                /*
+                 * Handles the bookings route for deleting bookings.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_rest_api_delete_booking_handler' => function(ContainerInterface $c) {
+                    return new DeleteBookingHandler($c->get('eddbk_bookings_controller'));
+                },
+
+                /*-------------------------------------------------------------*\
+                 * REST API route handlers - Clients                           *
+                \*-------------------------------------------------------------*/
+
+                /*
+                 * Handles the clients route that receives generic client queries.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_rest_api_query_clients_handler' => function (ContainerInterface $c) {
+                    return new QueryClientsHandler($c->get('eddbk_clients_controller'));
+                },
+
+                /*
+                 * Handles the clients route that receives generic client queries.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_rest_api_get_client_info_handler' => function (ContainerInterface $c) {
+                    return new ClientInfoHandler($c->get('eddbk_clients_controller'));
+                },
+
+                /*
+                 * Handles the clients route for creating new clients.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_rest_api_create_client_handler' => function (ContainerInterface $c) {
+                    return new CreateClientHandler($c->get('eddbk_clients_controller'), $c->get('eddbk_rest_api'));
+                },
+
+                /*-------------------------------------------------------------*\
+                 * Transformers - for data, not robots in disguise             *
+                \*-------------------------------------------------------------*/
+
+                /*
+                 * A no-operation transformer.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_noop_transformer' => function (ContainerInterface $c = null) {
+                    return new NoOpTransformer();
+                },
+
+                /*
+                 * The transformer that transforms bookings into the result that is send in REST API responses.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_rest_api_bookings_transformer' => function (ContainerInterface $c) {
+                    return new MapTransformer([
+                        [
+                            'source' => 'id',
```

Comment:
All these keys should use constants, such as [`MapTransformer::K_SOURCE`][]

[`MapTransformer::K_SOURCE`]: https://github.com/RebelCode/rcmod-eddbk-rest-api/blob/eda47e3ce60354e93e0b6cea654b5101a346edac/src/Transformer/MapTransformer.php#L152